RF24Network.cpp

```diff
@@ -1014,15 +1011,9 @@ bool ESBNetwork<radio_t>::write_to_pipe(uint16_t node, uint8_t pipe, bool multic
 {
     bool ok = false;
 
-    // Open the correct pipe for writing.
-    // First, stop listening so we can talk
-    if (!(networkFlags & FLAG_FAST_FRAG)) {
-        radio.stopListening();
-    }
-
     if (!(networkFlags & FLAG_FAST_FRAG) || (frame_buffer[6] == NETWORK_FIRST_FRAGMENT && networkFlags & FLAG_FAST_FRAG)) {
+        radio.stopListening(pipe_address(node, pipe));
```

Comment:
I see now why you wanted a `stopListening(uint64_t)` overload. `pipe_address()` returns an integer.

I look into migrating `pipe_address()` to return a buffer instead. I'll probably pass the address buffer by reference.